src/start.sh

```diff
@@ -75,10 +75,13 @@ start() {
         sleep 0.5
     done
 
-    #  Wait until the FTL log contains the "FTL started" message before continuing
-    while ! grep -q '########## FTL started' /var/log/pihole/FTL.log; do
-        sleep 0.5
-    done
+    # Wait until the FTL log contains the "FTL started" message before continuing, timeout after 10 seconds
+    # exit if we do not find it
+    pihole-FTL wait-for '########## FTL started' /var/log/pihole/FTL.log 10 "0 > /dev/null"
```

Comment:
On the PR for the shutdown test - you comment that we check for up to 60 secodns to give slow RISCV devices a chance to start up. Perhaps we should wait 60 seconds here.

Reply:
The difference is: Within the `start.sh, ` pihole-FTL wait-for` will only wait for the log line to appear, which is triggered by the ` capsh --user="${DNSMASQ_USER}" --keep=1 -- -c "/usr/bin/pihole-FTL $FTL_CMD >/dev/null" &` line just above. Not much time should be in between. 

The other PR watches the container from the outside before it kills the container. There are other (maintenance) tasks before FTL will even be invoked, e.g. running gravity. Therefore I gave it more time.